com.expidusos.file_manager.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.expidusos.file_manager
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: file_manager
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --socket=system-bus
+  - --socket=session-bus
+  - --filesystem=home
+modules:
+  - name: FileManager
+    buildsystem: simple
+    build-commands:
+      - cp -R bundle /app/expidus_file_manager
+      - mkdir -p /app/bin && ln -s /app/expidus_file_manager/file_manager /app/bin/file_manager
+      - install -D bundle/data/com.expidusos.file_manager.desktop /app/share/applications/com.expidusos.file_manager.desktop
+      - install -D bundle/data/com.expidusos.file_manager.metainfo.xml /app/share/metainfo/com.expidusos.file_manager.metainfo.xml
+      - install -D bundle/data/com.expidusos.file_manager.png /app/share/icons/hicolor/256x256/apps/com.expidusos.file_manager.png
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
```

Comment:
this does prevent building the flatpak on aarch64. Also it's very disappointing since the website say "for mobile devices"

Reply:
Flutter apps cannot be built in Flatpaks, I've tried and it's very difficult so I looked at how other flutter apps package themselves for Flatpaks. The app isn't mobile only, it's cross platform.

Reply:
and that's not what they do either.

https://github.com/flathub/flathub/wiki/App-Maintenance#flathubjson

Reply:
Oh I was looking at this https://github.com/vivian-ng/telsis_translator_flutter/blob/main/com.maplerain.telsis_translator_flutter.yml

Reply:
Alright, I've added a `flathub.json`.